boltdesignsystem/bolt
# DS-538: Update test banner

## Jira

https://pegadigitalit.atlassian.net/browse/DS-538

## Summary

Test was updated to the new schema.

## Details

Component was checked with master tickets.

## How to test

Check if it is up to date with standards mentioned in master ticket:

Refer to the Standarize Test ticket: https://pegadigitalit.atlassian.net/browse/DS-413
